src/test/java/com/codegym/jrugotom5/service/AdvertServiceTest.java

```diff
@@ -95,4 +97,39 @@ void testGetAdvertsByDateRange_InvalidDateRange() {
 
         assertEquals("'From' date should be after 'To' date.", exception.getMessage());
     }
+
+    @Test
+    void testGetAdvertsByCreateBy_WithAdverts_ReturnsDtoList() {
+        Long userId = 1L;
+        Advert advert1 = new Advert();
+        Advert advert2 = new Advert();
+        List<Advert> adverts = List.of(advert1, advert2);
+
+        when(advertRepository.getAdvertsByCreatedById(userId)).thenReturn(adverts);
+
+        AdvertInfoForCreatorDto dto1 = new AdvertInfoForCreatorDto();
+        AdvertInfoForCreatorDto dto2 = new AdvertInfoForCreatorDto();
+
+        when(modelMapper.map(advert1, AdvertInfoForCreatorDto.class)).thenReturn(dto1);
+        when(modelMapper.map(advert2, AdvertInfoForCreatorDto.class)).thenReturn(dto2);
+
+        List<AdvertInfoForCreatorDto> result = advertService.getAdvertsByUserId(userId);
+
+        assertEquals(2, result.size());
+        assertTrue(result.contains(dto1));
+        assertTrue(result.contains(dto2));
+        verify(advertRepository).getAdvertsByCreatedById(userId);
+    }
+
+    @Test
+    void testGetAdvertsByCreateBy_NoAdverts_ReturnsEmptyList() {
```

Comment:
IMO, there's no need to test for such case since you have input parameter validation on service layer. 
The DBMS just returns an empty list if "no records found" and that's Ok

Reply:
https://github.com/Volodymyr95/goto-m5f/pull/5/commits/d7c159416a5b898167525a7f2a110431f1e6344c#r1803620654